source/train/DescrptSeA.py

```diff
@@ -372,18 +383,26 @@ def _filter(self,
                                   self.filter_precision,
                                     tf.random_normal_initializer(stddev=stddev/np.sqrt(outputs_size[ii]+outputs_size[ii-1]), seed = seed), 
                                     trainable = trainable)
+                # only used when tensorboard was set as true
+                variable_summaries(w, 'matrix_'+str(ii)+'_'+str(type_i))
```

Comment:
I have refactored code according to PR #323 which indeed makes the implementation more conveniet. Also helper function `variable_summaries` was moved from `common.py` to `Network.py` since it is not needed anywhere else.